src/main/resources/static/js/common.js

```diff
@@ -239,25 +239,60 @@ function viewModel(backendInstanceData) {
 // End ## Load Back end list ##
 
 // Start ## Login and Security ##
-function doIfUserLoggedIn(user) {
-    setCurrentUser(user);
+function doIfUserLoggedIn(username) {
+    setCurrentUser(username);
+    showLoggedInUserInformation(username);
+    unlockSubscriptionButtons();
+}
+
+function showLoggedInUserInformation(username) {
     $("#userItem").show();
-    $("#userItem").addClass("user-login");
-    $("#ldapUserName").text(user);
+    $("#login-nav-bar-icon").addClass("user-login-icon");
+    $("#login-nav-bar-text").text(cropUsername(username));
+    $("#ldapUserName").text(username);
     $("#loginBlock").hide();
     $("#logoutBlock").show();
+}
+
+function unlockSubscriptionButtons() {
+    $(".logged-out-buttons").addClass("hidden_by_default");
+    $(".logged-in-buttons").removeClass("hidden_by_default");
     $(".show_if_authorized").prop('disabled', false);
+    $(".show_if_authorized").prop('title', "");
+}
+
+function cropUsername(username) {
```

Comment:
Ellipses can be added with css using:
overflow-x: hidden;
text-overflow: ellipsis;
You can't control the amount of characters but it does add ellipses if the text goes outside your side width.

Reply:
Yes, but it does not count how many characters has been given, it also requires a fixed width and it makes shorter names push notifications away.